.template/.github/workflows/test.yml

```diff
@@ -36,3 +36,12 @@ jobs:
 
       - name: Run widget tests, unit tests.
         run: flutter test --machine --coverage
+
+      - uses: codecov/codecov-action@v2
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }} # not required for public repos
+          files: ./coverage/lcov.info
+          flags: unittests # optional
+          name: Upload coverage to codecov
```

Comment:
@luongvo could you help to re-order `name` on top of others? 😄

Reply:
@manh-t done